### PR TITLE
Added basic JHU vs WHO DQ checker.

### PR DESCRIPTION
**Pull request by chrisvoncsefalvay**

### CODE REVIEW

**Comment by tfoldi** [comment]
let me have a look at this next week

**Comment by tfoldi** [comment]
I have a pending change to support notebooks without csv outputs. as soon as I merge it, I will merge this too

**Comment by chrisvoncsefalvay** [comment]
Thanks @tfoldi. Any way to have the images be pushed into `../docs` so that I can create a data quality monitor?

**Comment by tfoldi** [comment]
I will change the `s3_upload` step to upload all generated outputs to S3, not just CSVs. Then we can use the public S3 HTTP link to include it in the docs

File: notebooks/DQ_CHECKS.ipynb [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(21, 29.7))\n",
+    "plt.title(\"Spot plot, WHO vs JHU deaths\")\n",
+    "sns.set()\n",
+    "ax = sns.heatmap(jhu_who_delta_cases, annot=True, fmt=\".0f\", cmap=\"YlGnBu\")\n",
+    "ax.set_xticklabels(jhu_who_delta.loc[jhu_who_delta[\"Date\"] >= datetime.date(year=2020, month=3, day=20)][\"Date\"].dt.strftime('%d-%m-%Y'))\n",
+    "plt.xticks(rotation=90)\n",
+    "plt.savefig(output_folder + \"DQ_CHECKS_who_vs_jhu_deaths.png\")"
```
Comment (tfoldi) [inline notebooks/DQ_CHECKS.ipynb:193]: I just changed the output file to contain the notebook prefix

**Review by tfoldi — APPROVED** [review]

**Pull request merged**